src/editors/data/constants/tinyMCEStyles.js

```diff
@@ -1,214 +1,220 @@
-export default `@import url("https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;0,600;0,700;1,300;1,400;1,600;1,700&display=swap");
+const getStyles = () => (
```

Comment:
This file is also used in the Text Editor. Did you update that code as well?

Reply:
So to explain: the purpose of this was to allow you to call this with variables, because I thought I would need to work on the css here just for the question widget. As it turned out, I didn't need to, but just thought to leave this change in because it makes the css more flexible. Since I export default `getStyles()` in the bottom, my change is just a refactoring that has no consequences for the components that use it.

Reply:
You can keep it. I agree that it makes the css more flexible and readable.